File: TD/TD7/Code/2.cpp
/**
 * Remplir un tableau de taille 5 x 5, remplir par USER
 */
#include <iostream>

/* Procédure */

void remplirTab(float tab[5][5])
{
  for (int i = 0; i < 5; ++i)
    for (int j = 0; j < 5; ++j) {
      std::cout << "Donne moi une valeur pour tab [" << i << "]"
        << "[" << j << "] : " << std::flush;;
      std::cin >> tab[i][j];
    }
}

/* Utilisation */

int main()
{
  float tab[5][5];
  
  remplirTab(tab);
  
  return 0;
}

File: TD/TD7/Code/Python/2.py
#!/usr/bin/python
# Remplir un tableau de taille 5 x 5, remplir par USER


def remplir_tab(tab):
    for i in range(len(tab)):
        for j in range(len(tab[0])):
            print('Donne moi une valeur : ', end='')
            tab[i][j] = float(input())


tab = [[0 for x in range(5)] for y in range(5)]

remplir_tab(tab)
